tests/snippets/stdlib_socket.py

```diff
@@ -137,3 +137,24 @@
 
 with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
 	pass
+
+connector = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
You are not actually checking any functionality of the socket blocking / timeout feature. You can skip the blocking part as it requires some signal handling but all other options can be tested. Please tell me if you need help with defining the test cases.